config/initializers/secure_headers.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+SecureHeaders::Configuration.default do |config|
+  # rubocop:disable Lint/PercentStringArray
+  config.csp = {
+    base_uri: %w['self'],
+    default_src: %w['self'],
+    connect_src: %w['self' wss:],
+    font_src: %w['self' cdn.rawgit.com],
+    img_src: %w['self' data:],
+    object_src: %w['none'],
+    script_src: %w['self' 'unsafe-inline' 'unsafe-eval'],
```

Comment:
I tried to get rid of the `'unsafe-inline'` by using a nonce instead, but unfortunately it seems that we cannot add a nonce (that is respected) to cover the `onload` handler for our font preloading `<link>`. By using `'unsafe-inline'`, I'm favoring performance over security.